### PR TITLE
Multiple players moving in the appartment with third person camera

### PR DESCRIPTION
**Pull request by Arcod7**
Surprise, you are a rabbit :3

### CODE REVIEW

File: source/GEngine/libdev/systems/driver/output/RenderWindow.cpp
Comment (Popochounet) [inline source/GEngine/libdev/systems/driver/output/RenderWindow.cpp:1]: Remet LOG_WARNING

**Review by Popochounet — CHANGES_REQUESTED** [review]
Lapin zinzibar

**Comment by Nfire2103** [comment]
Pense au beautify ;)

**Review by Popochounet — APPROVED** [review]
top!

**Pull request merged**